simulation/stress_test/prepare_stress_test.py

```diff
@@ -0,0 +1,111 @@
+import sys
+sys.path.append("../../")
+import os
+import configparser
+import jupiter_config
+import logging
+import random
+import paramiko
+import socket
+import _thread
+import multiprocessing
+from multiprocessing import Process, Manager
+from flask import Flask, request
+
+
+
+logging.basicConfig(level = logging.INFO)
+app = Flask(__name__)
+
+# This script must run on XDC to put random stress test on random nodes in order to collect stats from DCOMP testbed 
+
+def get_nodes(node_info_file):
+  """read the node info from the file input
+  
+  Args:
+      node_info_file (str): path of ``node.txt``
+  
+  Returns:
+      dict: node information 
+  """
+  nodes = []
+  homes = []
+  node_file = open(node_info_file, "r")
+  for line in node_file:
+      node_line = line.strip().split(" ")
+      if node_line[0].startswith('home'): 
+        homes.append(node_line[1])
+      else:
+        nodes.append(node_line[1])
+  return homes, nodes
+
+def build_push_stress():
+    jupiter_config.set_globals()
+    print(jupiter_config.STRESS_IMAGE)
+    # Master node on DCOMP n0
+    ssh = connect_remote_ssh('n0')
```

Comment:
Sigh, this is a nightmare because we have to use an XDC. I don't like how we're using some arbitrary hostname here. It would be cleaner to get the master node's IP address from the jupiter_config.ini. This script is **way** too highly dependent on preconfiguration of DCOMP nodes. If this is absolutely necessary, then why don't we just stop using the XDC the second we are done with bootstrapping k8s? That is, these scripts should instead just run on the master node. There's a lot to discuss in this file. Let's do a code review over Zoom.